proc/syscall_windows.go

```diff
@@ -22,8 +22,53 @@ type _THREAD_BASIC_INFORMATION struct {
 	BasePriority   int32
 }
 
+type _CREATE_PROCESS_DEBUG_INFO struct {
```

Comment:
This format is my personal preference. Russ likes to follow Go convention here - for example DEBUG_EVENT becomes DebugEvent (or debugEvent if you want identifier to be non-exported). I don't like Russ's approach because:
- sometimes it creates name clashes - win32 might have DebugEvent function or something, also DebugEvent is a good name for something in you code (also CONTEXT -> Context for example), so you end-up thinking of a different name, so you end up with some completely different name;
- you loose ability to Google for original name - once you have changed DEBUG_EVENT into DebugEvent, how do you search for struct definition on the Net?
So I prefer to keep Microsoft names as is. I also add leading _ to make name not exported. We don't want github.com/derekparker/proc users to be able to access / see these names.